prismic-model/js/parts/article-body.js

```diff
@@ -294,6 +295,15 @@ export default {
           },
         },
       },
+      discussion: slice('Discussion', {
+        nonRepeat: {
+          title: heading('Title', 2),
+        },
+        repeat: {
+          speaker: link('Contributor', 'document', ['people']),
```

Comment:
Maybe we could keep this named contributor now that I see it in context here.

We're currently taking a contributor, parsing it as a `Person`, and calling it a `speaker`, which seems like unnecessary and potentially confusing specificity.